en/config/motors.md

```diff
@@ -0,0 +1,14 @@
+# Motor Checks
+
+After the airframe is setup and configured you should validate the motor assignment, spin direction and servo response:
+
+1. Remove all propellers, switch to Manual flight mode and arm the vehicle.
+1. Increase the throttle a bit, so the vehicle does not automatically disarm.
+   At the same time check that the motors respond to throttle changes.
+1. Check that all motors turn to the right direction according to the configured [airframe](../airframes/airframe_reference.md).
+   If any spin the wrong direction, they need to be reversed.
+   There are several options to achieve that:
+   - If using ESCs that support [DShot](../peripherals/dshot.md) you can reverse the direction via [DShot commands](../peripherals/dshot.md#commands).
```

Comment:
Note, I switched this over to be "easiest way first".  I haven't gone into detail on the other factors about using Dshot. IMO there are several reasons you might not want to use them too. I did state that there are other reasons you might prefer Dshot.

Reply:
Yes that's fine.